src/azure-cli/azure/cli/command_modules/identity/_help.py

```diff
@@ -0,0 +1,21 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps
+
+helps['identity'] = """
+type: group
+short-summary: Managed Service Identities
```

Comment:
@zhoxing-ms, we should update the doc because the term MSI is now deprecated. 

https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview

> Managed identities for Azure resources is the new name for the service formerly known as Managed Service Identity (MSI).

Reply:
OK~